### PR TITLE
Add manpage

### PR DESCRIPTION
**Pull request by mame98**
I added a simple manpage, that covers most general options and shows how to get the remaining information.

### CODE REVIEW

**Comment by CarterLi** [comment]
Great, this is what I really like.

File: doc/fastfetch.1.in [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+.SH OPTIONS
+
+.SS "Informative Options"
+.TP
+
+.B \-\-help \fI [command]
+Show help output, displaying all available options or help for given command
+.TP
+
+.B \-\-version
```
Comment (CarterLi) [inline doc/fastfetch.1.in:33]: We have `-v` as a shortcut

File: doc/fastfetch.1.in [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+.B \-\-list\-logos
+List available logos, they can be loaded with \fI \-\-logo
+.TP
+
+.B \-\-list-modules
+List available modules
+.TP
+
+.B \-\-list-presets
+List available presets, they can be loaded with \fI \-\-load\-config
```
Comment (CarterLi) [inline doc/fastfetch.1.in:54]: `--load-config` is deprecated. We should suggest `--config` or `-c` here

File: doc/fastfetch.1.in [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+.B \-\-list-features
+List the supported features fastfetch was compiled with
+.TP
+
+.B \-\-print-logos
+Print available logos
+.TP
+
+.B \-\-print\-default\-structure
```
Comment (CarterLi) [inline doc/fastfetch.1.in:65]: It should be `--print-structure`

File: doc/fastfetch.1.in [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+Print the default structure
+
+
+.SS "Display Options"
+.TP
+
+.B \-l, \-\-logo \fI <logo>
```
Comment (CarterLi) [inline doc/fastfetch.1.in:72]: We have `-l` as a shortcut

File: doc/fastfetch.1.in [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+Set the logo to display
+.TP
+
+.B \-c, \-\-structure \fI <structure>
```
Comment (CarterLi) [inline doc/fastfetch.1.in:76]: Should be `-s`

File: doc/fastfetch.1.in (hunk 1 above)
Comment (mame98) [reply doc/fastfetch.1.in]: fixed in 5445ef1581ab2cc8b5366fa49696225f555e3a61

File: doc/fastfetch.1.in (hunk 3 above)
Comment (mame98) [reply doc/fastfetch.1.in]: fixed in 5445ef1581ab2cc8b5366fa49696225f555e3a61

Also fixed help.txt in 1cd8235c2927ab0979197b14d3fecdf76fb9dec1

File: doc/fastfetch.1.in [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+.SS "Config Options"
+.TP
+
+.B \-\-print\-config\-system
+Print the default system config
+.TP
+
+.B \-\-print\-config\-user
+Print the default user config
+.TP
```
Comment (CarterLi) [inline doc/fastfetch.1.in:90]: These two options no longer works in dev branch, and about to be removed. Let's not add these.

File: doc/fastfetch.1.in (hunk 5 above)
Comment (mame98) [reply doc/fastfetch.1.in]: fixed in ec9bc9ba0235855ba50d9ff76d8c94385713ad37

File: doc/fastfetch.1.in (hunk 2 above)
Comment (mame98) [reply doc/fastfetch.1.in]: fixed in 5445ef1581ab2cc8b5366fa49696225f555e3a61

Also fixed help.txt in 1cd8235c2927ab0979197b14d3fecdf76fb9dec1

File: doc/fastfetch.1.in [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+To list all available modules, use \fB \-\-list\-modules \fR
+
+.SS "Config Files"
+
+fastfetch supports two types of config files. The legacy
+configuration files and a JSONC based format.
+
+A legacy fastfetch config file contains one command line flag on each
+line. If the line starts with an '#' it is ignored. The files must have
+the '.conf' extension.
```
Comment (CarterLi) [inline doc/fastfetch.1.in:123]: Since it's legacy, lets not add it here

File: doc/fastfetch.1.in (hunk 4 above)
Comment (mame98) [reply doc/fastfetch.1.in:72]: I think you missed the '-l' in line 72?

File: doc/fastfetch.1.in (hunk 6 above)
Comment (mame98) [reply doc/fastfetch.1.in]: Removed in a4c3d600a89a733aea33905c4b124e76fc1e5195

I also removed them from help.txt in e10fb1f17cc216a13be7f0eebb75b65b54aca455

File: doc/fastfetch.1.in (hunk 7 above)
Comment (mame98) [reply doc/fastfetch.1.in]: Removed in 7bb9baa92756ea01e0694a23b5f1a961e7f40ac4

**Comment by mame98** [comment]
Thank you for the in-depth review :+1:

File: doc/fastfetch.1.in [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+
+A JSONC config file is a JSON file that also supports comments with (//). Those
+files must have the extension '.jsonc'. You can migrate a legacy config
+file to the JSONC format by using \fB\-\-migrate\-config\fR.
```
Comment (CarterLi) [inline doc/fastfetch.1.in:115]: `--migrate-config` has been renamed to `--gen-config` in dev branch.

Comment (mame98) [reply doc/fastfetch.1.in]: fixed in cac0e57015326052647d422ec1b2b6893ce195d0

**Comment by CarterLi** [comment]
A few config related changes have been made recently in dev branch. You may check the [change log](https://github.com/fastfetch-cli/fastfetch/blob/dev/CHANGELOG.md)

File: doc/fastfetch.1.in [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+
+On successfull execution, fastfetch returns zero. If any error happened,
+the exit code will be non-zero.
+
+.SH OPTIONS
+
+.SS "Informative Options"
+.TP
+
+.B \-\-help \fI [command]
```
Comment (CarterLi) [inline doc/fastfetch.1.in:29]: `-h`

File: doc/fastfetch.1.in [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+
+.B \-\-list\-data\-paths
+List search paths of presets and logos
+.TP
+
+.B \-\-list\-logos
+List available logos, they can be loaded with \fI \-\-logo
+.TP
+
+.B \-\-list-modules
```
Comment (CarterLi) [inline doc/fastfetch.1.in:49]: Some `-`s are encoded; some are not. What's the difference?

File: doc/fastfetch.1.in (hunk 9 above)
Comment (mame98) [reply doc/fastfetch.1.in]: fixed in 310b3cc071dcd06e0feb6f01f977a8235cd10686

File: doc/fastfetch.1.in (hunk 10 above)
Comment (mame98) [reply doc/fastfetch.1.in]: I simply forgot to escape the other ones, fixed that in 8b6e7a6381acee0b8dde2201871a02d138b5725a

**Comment by CarterLi** [comment]
Thanks for contribution!

**Comment by mame98** [comment]
Thank you for reviewing and merging :+1:

**Pull request merged**